src/docs/install.md

```diff
@@ -23,19 +23,29 @@ in the root (ie, "buildroot") of your repo:
     :::bash
     curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants && touch pants.ini
 
+Before running the new `./pants` script, you should pin the Python version you want to run Pants with.
+To do this, add an entry like so to `pants.ini`, choosing between `2.7`, `3.6`, or `3.7`.
+
+    :::ini
+    [GLOBAL]
+    pants_engine_python_version: 3.6
+
+If you'd like to change which Python version Pants uses, just edit the entry in `pants.ini`.
+
```

Comment:
Question #1: do we want this to appear as the first command they do after `curl`? 

Pro: they don't bootstrap multiple virtual environments.
Con: maybe overwhelming for the user to immediately be confronted with this question, and better to get the program working with `./pants -V` and version pinned, *then* decide to pin interpreter if they want to.

Note you do not have to pin the interpreter—it will default to Python 2.7.